pkg/ccl/streamingccl/streamproducer/producer_job.go

```diff
@@ -84,12 +100,22 @@ func (p *producerJobResumer) Resume(ctx context.Context, execCtx interface{}) er
 			if err != nil {
 				return err
 			}
-			// The job completes successfully if the ingestion has been cut over.
-			if p := j.Progress(); p.GetStreamReplication().IngestionCutOver {
-				return nil
-			}
-			if isTimedOut(j) {
-				return errors.Errorf("replication stream %d timed out", p.job.ID())
+
+			prog := j.Progress()
```

Comment:
Should we also be checking for this condition at startup to avoid having to wait for one liveness tracking interval?